.github/workflows/ci.yml

```diff
@@ -7,40 +7,115 @@ on:
   pull_request:
 
 jobs:
-  test:
+  initial-code-check:
     runs-on: ubuntu-latest
 
     steps:
       - name: Check out
         uses: actions/checkout@v2
 
-      - name: Setup Scala
-        uses: olafurpg/setup-scala@v10
-        with:
-          java-version: adopt@1.11
-
-      - name: Run initial checks
+      - name: Run code checks
         run: |
           PATH=$PATH:./scripts
           set -x
           enforce-newline-at-eof
           prohibit-trailing-whitespace
+          enforce-issue-number-for-todos
 
           npm install --global remark-cli remark-lint-no-dead-urls remark-validate-links
           validate-markdown-links
 
+  code-style-check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.11
+
+      - name: Resolve dependencies
+        run: sbt update
+
+      - name: Check code style
+        # Curiously, scalafmtCheckAll doesn't cover *.sbt files, hence scalafmtSbtCheck is needed as well
+        run: sbt scalafmtCheckAll scalafmtSbtCheck "scalafixAll --check"
+
+  test-212:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.11
+
       - name: Resolve dependencies
         run: sbt update
 
       - name: Compile
         run: sbt compile Test/compile
 
       - name: Run tests
-        run: sbt test
+        run: |
+          sbt "++ 2.12.13 test"
+          sbt "++ 2.12.14 test"
+          sbt "++ 2.12.15 test"
 
-      - name: Check code style
-        # Curiously, scalafmtCheckAll doesn't cover *.sbt files, hence scalafmtSbtCheck is needed as well
-        run: sbt scalafmtCheckAll scalafmtSbtCheck "scalafixAll --check"
+  test-213-1:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.11
+
+      - name: Resolve dependencies
+        run: sbt update
+
+      - name: Compile
+        run: sbt compile Test/compile
+
+      - name: Run tests
+        run: |
+          sbt "++ 2.13.2 test"
+          sbt "++ 2.13.3 test"
+          sbt "++ 2.13.4 test"
+          sbt "++ 2.13.5 test"
+
+  test-213-2:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.11
+
+      - name: Resolve dependencies
+        run: sbt update
+
+      - name: Compile
+        run: sbt compile Test/compile
+
+      - name: Run tests
+        run: |
+          sbt "++ 2.13.6 test"
+          sbt "++ 2.13.7 test"
```

Comment:
Huh this one doesn't pass the CI, again due to scalafix... is it doable to make the version of `scalafix` sbt plugin (or the version of `semanticdb-scalac`?) conditional on `scala-version`?

Reply:
I put a condition in `build.sbt` file to fix that.